types/Keyguard.d.ts

```diff
@@ -5,8 +5,7 @@ interface Newable {
     new(...args: any[]): any
 }
 
-type DOMEvent = Event & {
-    target: Element
+interface Event {
     data: any
 }
```

Comment:
If we are adding `data` to an `Event`, why not also add `target: Element`?

Reply:
Because it's already defined in Event (as EventTarget | null).